crates/client/rpc/src/lib.rs

```diff
@@ -454,6 +454,17 @@ where
         Ok(Felt(chain_id.0))
     }
 
+    /// Checks if L1 Nonce has been used.
+    fn l1_nonce_unused(&self, nonce: u64) -> RpcResult<bool> {
+        let best_block_hash = self.client.info().best_hash;
+        let unused = self
```

Comment:
`is_unused`

Reply:
To be removed as https://github.com/keep-starknet-strange/madara/pull/1205#discussion_r1403780855